### PR TITLE
#28 - added tags to bottle descriptions

### PR DESCRIPTION
**Pull request by Erunno**

### CODE REVIEW

File: WhiskyMan.Backend/WhiskyMan.API/Controllers/BottlesController.cs [hunk 1]
```diff
@@ -24,17 +24,17 @@ public async Task<IActionResult> GetActiveBottlesForView()
...
+                new {Id= 1, Name ="bottleName 1", Distillery = "distillery 1", ShotPrice = 45, PictureUrl = "https://bottlesandstories.cz/1245-large_default/glenfiddich-12-yo-07l-40.jpg", Tags=new List<string> { "smoky", "cherry", "advanced" } },
+                new {Id= 1, Name ="bottleName 1", Distillery = "distillery 1", ShotPrice = 45, PictureUrl = "https://bottlesandstories.cz/1245-large_default/glenfiddich-12-yo-07l-40.jpg", Tags=new List<string> { "smoky", "cherry", "advanced" } },
+                new {Id= 1, Name ="bottleName 1", Distillery = "distillery 1", ShotPrice = 45, PictureUrl = "https://bottlesandstories.cz/1245-large_default/glenfiddich-12-yo-07l-40.jpg", Tags=new List<string> { "smoky", "cherry", "advanced" } },
+                new {Id= 1, Name ="bottleName 1", Distillery = "distillery 1", ShotPrice = 45, PictureUrl = "https://bottlesandstories.cz/1245-large_default/glenfiddich-12-yo-07l-40.jpg", Tags = new List<string> { "smoky", "cherry", "advanced" } },
             });
         }
```
Comment (Erunno) [inline WhiskyMan.Backend/WhiskyMan.API/Controllers/BottlesController.cs:40]: Temporary
The endpoint will be implemented in another issue

File: WhiskyMan.Backend/WhiskyMan.API/Migrations/20210104171640_Init.Designer.cs [hunk 2]
```diff
@@ -9,7 +9,7 @@
 namespace WhiskyMan.API.Migrations
```
Comment (Erunno) [inline WhiskyMan.Backend/WhiskyMan.API/Migrations/20210104171640_Init.Designer.cs:9]: Don't bother reading whatever is in folder Migrations (it is generated).

File: WhiskyMan.Backend/WhiskyMan.Models/BottleDescription/TagReference.cs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WhiskyMan.Models.BottleDescription
+{
+    public record TagReference
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
```
Comment (Erunno) [inline WhiskyMan.Backend/WhiskyMan.Models/BottleDescription/TagReference.cs:12]: By (my) convention:
Every object which has suffix `Reference` has to have at least field `id` and `name` (may have others if it makes sense e.g. `pictureUrl` but not something too specific e.g. `drunkBottlesLastYear`)

**Review by Jokekill — APPROVED** [review]
All seems good!

**Pull request merged**